R/derive_vars_dt.R

```diff
@@ -433,6 +417,7 @@ impute_dtc_dt <- function(dtc,
                           min_dates = NULL,
                           max_dates = NULL,
                           preserve = FALSE) {
+  is_datetime <- FALSE
```

Comment:
I don't think we need a variable here because we will never change its value. Thus the value can directly passed to the helper functions where needed.